lib/addons/fluxcd/index.ts

```diff
@@ -9,6 +9,51 @@ import { FluxGitRepository } from "./gitrepository";
 import { FluxKustomization } from "./kustomization";
 import { KubernetesManifest } from 'aws-cdk-lib/aws-eks/lib/k8s-manifest';
 
+/**
+ * Options for a FluxCD repository
+ */
+export interface FluxCDGitRepo extends Omit<spi.GitOpsApplicationDeployment, "repository">{

```

Comment:
we probably need to review namings and make consistent choices, e.g. flux vs fluxcd. also do we want the flux prefix on kustomizations? then use everywhere... e.g. I see kustomizations and fluxKustomizationPath and fluxPrune in Kustomization props...

Reply:
Flux in the name of the interface is enough to remove the prefix from each field.  The interfaces should be publicly exported, and if they are part of the addon contract, they should stay with the add-on.